README.md

```diff
@@ -62,8 +77,8 @@ plugin as a Packer integration refer to the [Developing Plugins](https://develop
 
 # Requirements
 
--	[packer-plugin-sdk](https://github.com/hashicorp/packer-plugin-sdk) >= v0.2.9
+-	[packer-plugin-sdk](https://github.com/hashicorp/packer-plugin-sdk) >= v0.5.2
```

Comment:
Are the SDK references in `go.mod`/`go.sum` up-to-date also?

Reply:
Yes - we bumped these when you added the integration framework configuration bits.